docs/lesson-plans/week-7.md

```diff
@@ -0,0 +1,632 @@
+    FROM table_name
+-- Note how equality comparison is done with a single "=". Even for string types.
+    WHERE names = 'someName'
+-- Can do comparisons with numbers.
+        AND column > 5
+-- Can do comparisons with dates too! Also, you can group conditions together if you want to using parentheses.
+        OR (some_date_column_too BEFORE CURRENT_TIMESTAMP
+-- Assuming the 'you' column is a VARCHAR/CHAR/TEXT type, you can search for strings that start with 'hello' using the 
+-- 'like' operator. This will find results like 'hello', 'hello ', and 'hello world'. The '%' is like a wildcard, it can
+-- match any string when used with like.
+            AND you LIKE "hello%")
+-- Check for null/not null with 'IS NULL' and 'IS NOT NULL'.
+        AND want IS NULL;
+    
+```
+
+An example using the `people` table:
+```sql
+-- Select all fields from every record in people.
+SELECT * FROM people;
+
+-- Select all fields from every record in people who are less than 20.
+SELECT * FROM people 
+    WHERE age < 20;
+
+-- Select the name and id fields (in that order) from every record where the name is 'like' jump.
+SELECT name, id FROM people 
+    WHERE name LIKE 'jump%';
+```
+
+#### The UPDATE Statement
+
+The `UPDATE` statement is the statement used for altering preexisting data (CR**U**D). The structure is somewhat 
+similar to the previous statements, so we'll just show you an example of how it works.
+
+Example of update statements:
+```sql
+-- Update columns on the given table name
+UPDATE table_name
+-- Set the column 'column_name' to be some calculation or data. The "=" sign can be used as both assignment and 
+-- comparison.
+    SET column_name = calculation_or_data;
+
+UPDATE table_name
+-- Set columns 1 and 2 to the numbers 1 and 2.
+    SET column1_name = 1, 
+        column2_name = 2,
+-- Set column 3 to the sum of 1 and 2 (yes, you can use the values of columns on the current table to set values -- it 
+-- uses the value before updating column1_name and column2_name though).
+        column3_name = column1_name + column2_name;
+
+UPDATE table_name 
+    SET value = "Hello Jumpstarters!"
+-- A query like the ones above but it only activates on a row with a certain condition.
+    WHERE condition;
+```
+
+An example using the `people` table:
+```sql
+-- Set all signup_date values to now.
+UPDATE people
+    SET signup_date = CURRENT_TIMESTAMP;
+
+-- Set all ages to the user's id.
+UPDATE people
+    SET age = id;
+
+-- Set all bio values to be the person's name and set the names to be "Jumpstarter".
+UPDATE people
+    SET bio = name,
+        name = "Jumpstarter";
+```
+
+#### The DELETE Statement
+
+The `DELETE` statement is the statement used for deleting existing data (CRU**D**). Again, the structure is similar
+to the previous statements. 
+
+>Note: if you want to delete everything from the table without having to drop and recreate 
+>the table, you can use the `TRUNCATE table_name;` statement instead. 
+
```

Comment:
maybe add another note for when the table connects to another one - to make sure the data is safe deleted (not sure if this is something that will be covered in the later lectures)

Reply:
I think this might be something more for the following lesson. Thoughts?

Reply:
sounds good 💯